Builder/ElasticSelectQueryBuilder.php

```diff
@@ -58,11 +58,14 @@ public function buildElasticQueryFromGeneric(
             } else {
                 $fieldsData = [];
             }
+
+            $searchTerm = addcslashes(strval($genericQuery->getSearchTerm()), '+-&|!(){}[]^”~*?:\\/');
```

Comment:
I'm not sure this is _quite_ correct - the characters are listed here https://lucene.apache.org/core/2_9_4/queryparsersyntax.html#Escaping%20Special%20Characters and it should be noted that there are some grouped characters there. Specifically, including `\` in the charlist there may make for some double-escaping. However, it is a step forward as stands.